shell/app/modules/project/common/components/issue/issue-drawer.scss

```diff
@@ -33,6 +33,104 @@
       margin-left: 12px;
       font-size: 16px;
     }
+
+    .followed {
+      color: $color-info;
+      vertical-align: middle;
+      cursor: pointer;
+      margin: 3px 0 0 0;
```

Comment:
why 3px? it is from UED?

Reply:
Because the selected and unselected Icon are not the same, in order to keep them in the same position when switching.